_conferences/2025/01_index.html

```diff
@@ -15,7 +15,7 @@
 ---
 <div class="mec2025-page">
 
-    <p><em>Update (2024-10-28):</em> Submission phase has opened. Please see our <a href="conference/2025/call">Call for Proposals</a> for further details.</p>
```

Comment:
@annplaksin This should be `href="/conference/2025/call/"` (note the slash at the beginning), otherwise it will redirect to the 2026 call next year (and similar in all upcoming years) ;)

Reply:
So it didn't work because of the slash... 🙄 
Thanks!

Reply:
But you know, you could have changed it here yourself and not just complaining

Reply:
Yes, I know ;) Helping people to help themselves :D